### ISSUE TITLE
[1.16.3] Rendering Overlay 

### ISSUE THREAD

**Issue by Rukia1727** [ShetiPhian/Localizations-and-Issues#271]
Hasn't done this until I installed these eight mods : 
dotHack-1.16.3-1.0.0-a
firstaid-1.9.4
CookingForBlockheads_1.16.3-9.2.1
BetterMineshaftsForge-1.16.2-1.0
DoggyTalents-1.16.3-2.0.0.3
Valheisia_structures-16.0.3
simplefarming-1.16.3-1.3.3

The other two instances have Valhesia. I'm going to remove it and see if that fixes it

---- Minecraft Crash Report ----
// Daisy, daisy...

Time: 10/10/20 1:52 PM
Description: Rendering overlay

java.lang.IllegalStateException: Failed to load foliage color texture 'terraqueous:textures/colormap/lightvibrant.png'
	at shetiphian.terraqueous.client.resources.FoliageColorReloadListener.get(FoliageColorReloadListener.java:42) ~[?:1.7.3] {re:classloading,pl:runtimedistcleaner:A}
	at shetiphian.terraqueous.client.resources.FoliageColorReloadListener.prepare(FoliageColorReloadListener.java:30) ~[?:1.7.3] {re:classloading,pl:runtimedistcleaner:A}
	at shetiphian.terraqueous.client.resources.FoliageColorReloadListener.func_212854_a_(FoliageColorReloadListener.java:15) ~[?:1.7.3] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.resources.ReloadListener.func_215270_b(SourceFile:11) ~[?:?] {re:classloading}
	at net.minecraft.client.resources.ReloadListener$$Lambda$6805/1158327119.get(Unknown Source) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1582) ~[?:1.8.0_51] {}
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1574) ~[?:1.8.0_51] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) ~[?:1.8.0_51] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056) ~[?:1.8.0_51] {}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1689) ~[?:1.8.0_51] {re:computing_frames}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157) ~[?:1.8.0_51] {}
Caused by: java.io.FileNotFoundException: terraqueous:textures/colormap/lightvibrant.png
	at net.minecraft.resources.SimpleReloadableResourceManager.func_199002_a(SimpleReloadableResourceManager.java:63) ~[?:?] {re:classloading}
	at net.minecraft.client.resources.ColorMapLoader.func_217820_a(SourceFile:14) ~[?:?] {re:classloading}
	at shetiphian.terraqueous.client.resources.FoliageColorReloadListener.get(FoliageColorReloadListener.java:40) ~[?:1.7.3] {re:classloading,pl:runtimedistcleaner:A}
	... 10 more

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.client.renderer.GameRenderer.func_195458_a(GameRenderer.java:807) ~[?:?] {re:classloading,pl:accesstransformer:B,xf:OptiFine:default}
-- Overlay render details --
Details:
	Overlay name: net.minecraft.client.gui.ResourceLoadProgressGui
Stacktrace:
	at net.minecraft.client.renderer.GameRenderer.func_195458_a(GameRenderer.java:807) ~[?:?] {re:classloading,pl:accesstransformer:B,xf:OptiFine:default}
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:953) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:584) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:184) [?:?] {re:classloading,pl:runtimedistcleaner:A}
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_51] {}
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_51] {}
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_51] {}
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_51] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51) [forge-1.16.3-34.1.5.jar:34.1] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider$$Lambda$477/1994249754.call(Unknown Source) [forge-1.16.3-34.1.5.jar:34.1] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:81) [modlauncher-7.0.1.jar:?] {re:classloading}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:65) [modlauncher-7.0.1.jar:?] {re:classloading}

-- System Details --
Details:
	Minecraft Version: 1.16.3
	Minecraft Version ID: 1.16.3
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 16441666272 bytes (15679 MB) / 21340618752 bytes (20352 MB) up to 21474836480 bytes (20480 MB)
	CPUs: 8
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx20G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	ModLauncher: 7.0.1+78+master.e9771d8
	ModLauncher launch target: fmlclient
	ModLauncher naming: srg
	ModLauncher services: 
		/mixin-0.8.1.jar mixin PLUGINSERVICE 
		/eventbus-3.0.3-service.jar eventbus PLUGINSERVICE 
		/forge-1.16.3-34.1.5.jar object_holder_definalize PLUGINSERVICE 
		/forge-1.16.3-34.1.5.jar runtime_enum_extender PLUGINSERVICE 
		/accesstransformers-2.2.0-shadowed.jar accesstransformer PLUGINSERVICE 
		/forge-1.16.3-34.1.5.jar capability_inject_definalize PLUGINSERVICE 
		/forge-1.16.3-34.1.5.jar runtimedistcleaner PLUGINSERVICE 
		/mixin-0.8.1.jar mixin TRANSFORMATIONSERVICE 
		/OptiFine_1.16.3_HD_U_G3.jar OptiFine TRANSFORMATIONSERVICE 
		/forge-1.16.3-34.1.5.jar fml TRANSFORMATIONSERVICE 
	FML: 34.1
	Forge: net.minecraftforge:34.1.5
	FML Language Providers: 
		javafml@34.1
		minecraft@1
	Mod List: 
		vanillaextension-1.16.3-1.0.10.jar Vanilla Extension {vanillaextension@1.16.3-1.0.10 COMMON_SETUP}
		Ma-Enchants-1.16.3-3.4.0.jar Ma Enchants {ma-enchants@1.16.3-3.4.0 COMMON_SETUP}
		dotHack-1.16.3-1.0.0-a.jar AzureDoom dotHack Mod {dothack@1.0.0-a VALIDATE}
		Variant16x-1.16.3-v0.15.4.2_beta.jar Variant16x {variant16x@0.15.4.2_beta COMMON_SETUP}
		simplemagnets-1.0.8-mc1.16.3.jar Simple Magnets {simplemagnets@1.0.8 COMMON_SETUP}
		treeharvester_1.16.3-1.6.jar Tree Harvester {treeharvester@1.6 COMMON_SETUP}
		DoggyTalents-1.16.3-2.0.0.3.jar Doggy Talents 2 {doggytalents@2.0.0.3 COMMON_SETUP}
		placeableblazerods_1.16.3_1.0.jar Placeable Blaze Rods {placeableblazerods@1.0 COMMON_SETUP}
		ChineseWorkshop-1.16.3-3.2.0.jar ChineseWorkshop {chineseworkshop@3.2.0 COMMON_SETUP}
		Kiwi-1.16.3-3.4.0.jar Kiwi {kiwi@3.4.0 COMMON_SETUP}
		mcw-windows-1.0.2-mc1.16.3.jar Macaw's Windows {mcwwindows@1.0.2 COMMON_SETUP}
		WallpaperCraft-1.16.3-1.4.5.jar Wallpapercraft {wallpapercraft@1.4.5 COMMON_SETUP}
		WallpapercraftSlabs-1.16.3-1.4.0.jar Wallpapercraft - Slabs! {wallpapercraftslabs@1.4.0 COMMON_SETUP}
		ForgeEndertech-1.16.3-7.0.4.0-build.0017.jar Forge Endertech {forgeendertech@7.0.4.0 COMMON_SETUP}
		WallpapercraftStairs-1.16.3-1.4.0.jar Wallpapercraft - Stairs {wallpapercraftstairs@1.4.0 COMMON_SETUP}
		BasaltBlocks-1.16.3-1.0.0.jar Basalt Blocks {basaltblocks@1.0.0 COMMON_SETUP}
		CookingForBlockheads_1.16.3-9.2.1.jar Cooking for Blockheads {cookingforblockheads@9.2.1 COMMON_SETUP}
		[1.16.3]-additionalbars-1.5.2.jar Additional Bars {additionalbars@1.5.2 COMMON_SETUP}
		Machines-1.16.3-1.7.0.0.jar SimpleOres:Machines {simple_machines@1.16.3-1.7.0.0 COMMON_SETUP}
		jei-1.16.3-7.3.2.36.jar Just Enough Items {jei@7.3.2.29 COMMON_SETUP}
		invtweaks-1.16.2-1.0.0.jar Inventory Tweaks Renewed {invtweaks@1.16.2-1.0.0 COMMON_SETUP}
		u_team_core-1.16.3-3.1.9.181.jar U Team Core {uteamcore@3.1.9.181 COMMON_SETUP}
		mcw-doors-1.0.0-mc1.16.3.jar Macaw's Doors {mcwdoors@1.0.0 COMMON_SETUP}
		firstaid-1.9.4.jar First Aid {firstaid@1.9.4 COMMON_SETUP}
		forge-1.16.3-34.1.5-universal.jar Forge {forge@34.1.5 COMMON_SETUP}
		shetiphiancore-1.16-3.8.2.jar ShetiPhian-Core {shetiphiancore@3.8.2 COMMON_SETUP}
		RevampedWolf-1.16.3-0.6.2.jar RevampedWolf {revampedwolf@1.16.3-0.6.2 COMMON_SETUP}
		SimpleOres2-1.16.3-2.3.0.2.jar Simple Ores {simpleores@1.16.3-2.3.0.2 COMMON_SETUP}
		forge-1.16.3-34.1.5-client.jar Minecraft {minecraft@1.16.3 COMMON_SETUP}
		useful_backpacks-1.16.3-1.11.5.82.jar Useful Backpacks {usefulbackpacks@1.11.5.82 COMMON_SETUP}
		DustrialDecor-1.0.6-1.16.3.jar 'Dustrial Decor {dustrial_decor@1.0.6 COMMON_SETUP}
		MouseTweaks-2.13-mc1.16.2.jar Mouse Tweaks {mousetweaks@2.13 COMMON_SETUP}
		assortedstorage-1.16.3-1.0.1.jar Assorted Storage {assortedstorage@1.16.3-1.0.1 COMMON_SETUP}
		BiomesOPlenty-1.16.3-12.0.0.400-universal.jar Biomes O' Plenty {biomesoplenty@1.16.3-12.0.0.400 COMMON_SETUP}
		valhelsia_structures-16.0.3.jar Valhelsia Structures {valhelsia_structures@16.0.3 COMMON_SETUP}
		LetSleepingDogsLie-1.16.3-1.1.1.jar Let Sleeping Dogs Lie {dogslie@1.1.1 COMMON_SETUP}
		SimpleCoreLib-1.16.3-2.0.0.5.jar SimpleCoreLib {simplecorelib@1.16.3-2.0.0.5 COMMON_SETUP}
		ornamental-1.16.3-4.2.jar Ornamental {ornamental@4.2 COMMON_SETUP}
		AdLods-1.16.3-4.0.2.0-build.0017.jar Large Ore Deposits {adlods@4.0.2.0 COMMON_SETUP}
		masonry-1.16.3-0.3.6.jar Masonry {masonry@NONE COMMON_SETUP}
		simplylight-1.16.3-1.0.2.jar Simply Light {simplylight@1.16.3-1.0.2 COMMON_SETUP}
		simplefarming-1.16.3-1.3.3.jar Simple Farming {simplefarming@1.16.3-1.3.3 COMMON_SETUP}
		xercamod-1.16.3-1.0.jar Xerca Mod {xercamod@1.16.3-1.0 COMMON_SETUP}
		WallpapercraftWalls-1.16.3-1.4.0.jar Wallpapercraft - Walls! {wallpapercraftwalls@1.4.0 COMMON_SETUP}
		collective-1.16.3-1.45.jar Collective {collective@1.45 COMMON_SETUP}
		DoubleSlabs-1.16-3.4.10.jar Double Slabs {doubleslabs@3.4.10 COMMON_SETUP}
		ftb-ultimine-1.1.0.10.jar FTB Ultimine {ftbultimine@1.1.0.10 COMMON_SETUP}
		buildersaddition-1.16.3-20200930a.jar Builders Crafts & Addition {buildersaddition@1.16.3-20200930a COMMON_SETUP}
		worldedit-forge-mc1.16.3-7.2.0-beta-05.jar WorldEdit {worldedit@7.2.0-beta-05+20b3da4 COMMON_SETUP}
		mcws-roofs-1.16.3-v2.0.0.jar Macaw's Roofs {mcwroofs@2.0.0 COMMON_SETUP}
		Aesthetics-1.16.3-1.10.0.0.jar Simple Ores: Aesthetics {aesthetics@1.16.3-1.10.0.0 COMMON_SETUP}
		WoV-1.1.4.jar War of Valor {shocked@1.16.3-1.1.4 COMMON_SETUP}
		ilikewood-1.16.3-4.0.2.2.jar I Like Wood {ilikewood@1.16.3-4.0.2.2 COMMON_SETUP}
		byg-1.0.7.jar Oh The Biomes You'll Go {byg@1.0.7 COMMON_SETUP}
		Aquaculture-1.16.3-2.1.1.jar Aquaculture 2 {aquaculture@1.16.3-2.1.1 COMMON_SETUP}
		AdChimneys-1.16.3-6.0.2.0-build.0017.jar Advanced Chimneys {adchimneys@6.0.2.0 COMMON_SETUP}
		terraqueous-1.16-1.7.3.jar Terraqueous {terraqueous@1.7.3 COMMON_SETUP, terraheads@1.0.0 COMMON_SETUP, pixelpowers@2.0.0 COMMON_SETUP}
		BetterMineshaftsForge-1.16.2-1.0.jar YUNG's Better Mineshafts {bettermineshafts@1.16.2-1.0 COMMON_SETUP}
	Kiwi Modules: 
		chineseworkshop:blocks
		chineseworkshop:debug_stick
		chineseworkshop:decorations
		chineseworkshop:materials
		chineseworkshop:retexture
		kiwi:contributors
		kiwi:data
	Launched Version: 1.16.3-forge-34.1.5
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: GeForce GTX 1060 6GB/PCIe/SSE2 GL version 4.6.0 NVIDIA 445.75, NVIDIA Corporation
	GL Caps: Using framebuffer using OpenGL 3.0
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'
	Type: Client (map_client.txt)
	Graphics mode: fast
	Resource Packs: 
	Current Language: English (US)
	CPU: 8x AMD FX-8370 Eight-Core Processor 
	OptiFine Version: OptiFine_1.16.3_HD_U_G3
	OptiFine Build: 20200916-082411
	Render Distance Chunks: 10
	Mipmaps: 0
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: null
	OpenGlVersion: 4.6.0 NVIDIA 445.75
	OpenGlRenderer: GeForce GTX 1060 6GB/PCIe/SSE2
	OpenGlVendor: NVIDIA Corporation
	CpuCount: 8

**Comment by Rukia1727** [comment]
Hasn't done this until I installed these eight mods : 
dotHack-1.16.3-1.0.0-a
firstaid-1.9.4
CookingForBlockheads_1.16.3-9.2.1
BetterMineshaftsForge-1.16.2-1.0
DoggyTalents-1.16.3-2.0.0.3
Valheisia_structures-16.0.3
simplefarming-1.16.3-1.3.3

The other two instances have Valhesia. I'm going to remove it and see if that fixes it

---- Minecraft Crash Report ----
// Daisy, daisy...

Time: 10/10/20 1:52 PM
Description: Rendering overlay

java.lang.IllegalStateException: Failed to load foliage color texture 'terraqueous:textures/colormap/lightvibrant.png'
	at shetiphian.terraqueous.client.resources.FoliageColorReloadListener.get(FoliageColorReloadListener.java:42) ~[?:1.7.3] {re:classloading,pl:runtimedistcleaner:A}
	at shetiphian.terraqueous.client.resources.FoliageColorReloadListener.prepare(FoliageColorReloadListener.java:30) ~[?:1.7.3] {re:classloading,pl:runtimedistcleaner:A}
	at shetiphian.terraqueous.client.resources.FoliageColorReloadListener.func_212854_a_(FoliageColorReloadListener.java:15) ~[?:1.7.3] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.resources.ReloadListener.func_215270_b(SourceFile:11) ~[?:?] {re:classloading}
	at net.minecraft.client.resources.ReloadListener$$Lambda$6805/1158327119.get(Unknown Source) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1582) ~[?:1.8.0_51] {}
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1574) ~[?:1.8.0_51] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) ~[?:1.8.0_51] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056) ~[?:1.8.0_51] {}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1689) ~[?:1.8.0_51] {re:computing_frames}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157) ~[?:1.8.0_51] {}
Caused by: java.io.FileNotFoundException: terraqueous:textures/colormap/lightvibrant.png
	at net.minecraft.resources.SimpleReloadableResourceManager.func_199002_a(SimpleReloadableResourceManager.java:63) ~[?:?] {re:classloading}
	at net.minecraft.client.resources.ColorMapLoader.func_217820_a(SourceFile:14) ~[?:?] {re:classloading}
	at shetiphian.terraqueous.client.resources.FoliageColorReloadListener.get(FoliageColorReloadListener.java:40) ~[?:1.7.3] {re:classloading,pl:runtimedistcleaner:A}
	... 10 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.client.renderer.GameRenderer.func_195458_a(GameRenderer.java:807) ~[?:?] {re:classloading,pl:accesstransformer:B,xf:OptiFine:default}
-- Overlay render details --
Details:
	Overlay name: net.minecraft.client.gui.ResourceLoadProgressGui
Stacktrace:
	at net.minecraft.client.renderer.GameRenderer.func_195458_a(GameRenderer.java:807) ~[?:?] {re:classloading,pl:accesstransformer:B,xf:OptiFine:default}
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:953) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:584) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:184) [?:?] {re:classloading,pl:runtimedistcleaner:A}
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_51] {}
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_51] {}
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_51] {}
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_51] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51) [forge-1.16.3-34.1.5.jar:34.1] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider$$Lambda$477/1994249754.call(Unknown Source) [forge-1.16.3-34.1.5.jar:34.1] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:81) [modlauncher-7.0.1.jar:?] {re:classloading}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:65) [modlauncher-7.0.1.jar:?] {re:classloading}


-- System Details --
Details:
	Minecraft Version: 1.16.3
	Minecraft Version ID: 1.16.3
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 16441666272 bytes (15679 MB) / 21340618752 bytes (20352 MB) up to 21474836480 bytes (20480 MB)
	CPUs: 8
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx20G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	ModLauncher: 7.0.1+78+master.e9771d8
	ModLauncher launch target: fmlclient
	ModLauncher naming: srg
	ModLauncher services: 
		/mixin-0.8.1.jar mixin PLUGINSERVICE 
		/eventbus-3.0.3-service.jar eventbus PLUGINSERVICE 
		/forge-1.16.3-34.1.5.jar object_holder_definalize PLUGINSERVICE 
		/forge-1.16.3-34.1.5.jar runtime_enum_extender PLUGINSERVICE 
		/accesstransformers-2.2.0-shadowed.jar accesstransformer PLUGINSERVICE 
		/forge-1.16.3-34.1.5.jar capability_inject_definalize PLUGINSERVICE 
		/forge-1.16.3-34.1.5.jar runtimedistcleaner PLUGINSERVICE 
		/mixin-0.8.1.jar mixin TRANSFORMATIONSERVICE 
		/OptiFine_1.16.3_HD_U_G3.jar OptiFine TRANSFORMATIONSERVICE 
		/forge-1.16.3-34.1.5.jar fml TRANSFORMATIONSERVICE 
	FML: 34.1
	Forge: net.minecraftforge:34.1.5
	FML Language Providers: 
		javafml@34.1
		minecraft@1
	Mod List: 
		vanillaextension-1.16.3-1.0.10.jar Vanilla Extension {vanillaextension@1.16.3-1.0.10 COMMON_SETUP}
		Ma-Enchants-1.16.3-3.4.0.jar Ma Enchants {ma-enchants@1.16.3-3.4.0 COMMON_SETUP}
		dotHack-1.16.3-1.0.0-a.jar AzureDoom dotHack Mod {dothack@1.0.0-a VALIDATE}
		Variant16x-1.16.3-v0.15.4.2_beta.jar Variant16x {variant16x@0.15.4.2_beta COMMON_SETUP}
		simplemagnets-1.0.8-mc1.16.3.jar Simple Magnets {simplemagnets@1.0.8 COMMON_SETUP}
		treeharvester_1.16.3-1.6.jar Tree Harvester {treeharvester@1.6 COMMON_SETUP}
		DoggyTalents-1.16.3-2.0.0.3.jar Doggy Talents 2 {doggytalents@2.0.0.3 COMMON_SETUP}
		placeableblazerods_1.16.3_1.0.jar Placeable Blaze Rods {placeableblazerods@1.0 COMMON_SETUP}
		ChineseWorkshop-1.16.3-3.2.0.jar ChineseWorkshop {chineseworkshop@3.2.0 COMMON_SETUP}
		Kiwi-1.16.3-3.4.0.jar Kiwi {kiwi@3.4.0 COMMON_SETUP}
		mcw-windows-1.0.2-mc1.16.3.jar Macaw's Windows {mcwwindows@1.0.2 COMMON_SETUP}
		WallpaperCraft-1.16.3-1.4.5.jar Wallpapercraft {wallpapercraft@1.4.5 COMMON_SETUP}
		WallpapercraftSlabs-1.16.3-1.4.0.jar Wallpapercraft - Slabs! {wallpapercraftslabs@1.4.0 COMMON_SETUP}
		ForgeEndertech-1.16.3-7.0.4.0-build.0017.jar Forge Endertech {forgeendertech@7.0.4.0 COMMON_SETUP}
		WallpapercraftStairs-1.16.3-1.4.0.jar Wallpapercraft - Stairs {wallpapercraftstairs@1.4.0 COMMON_SETUP}
		BasaltBlocks-1.16.3-1.0.0.jar Basalt Blocks {basaltblocks@1.0.0 COMMON_SETUP}
		CookingForBlockheads_1.16.3-9.2.1.jar Cooking for Blockheads {cookingforblockheads@9.2.1 COMMON_SETUP}
		[1.16.3]-additionalbars-1.5.2.jar Additional Bars {additionalbars@1.5.2 COMMON_SETUP}
		Machines-1.16.3-1.7.0.0.jar SimpleOres:Machines {simple_machines@1.16.3-1.7.0.0 COMMON_SETUP}
		jei-1.16.3-7.3.2.36.jar Just Enough Items {jei@7.3.2.29 COMMON_SETUP}
		invtweaks-1.16.2-1.0.0.jar Inventory Tweaks Renewed {invtweaks@1.16.2-1.0.0 COMMON_SETUP}
		u_team_core-1.16.3-3.1.9.181.jar U Team Core {uteamcore@3.1.9.181 COMMON_SETUP}
		mcw-doors-1.0.0-mc1.16.3.jar Macaw's Doors {mcwdoors@1.0.0 COMMON_SETUP}
		firstaid-1.9.4.jar First Aid {firstaid@1.9.4 COMMON_SETUP}
		forge-1.16.3-34.1.5-universal.jar Forge {forge@34.1.5 COMMON_SETUP}
		shetiphiancore-1.16-3.8.2.jar ShetiPhian-Core {shetiphiancore@3.8.2 COMMON_SETUP}
		RevampedWolf-1.16.3-0.6.2.jar RevampedWolf {revampedwolf@1.16.3-0.6.2 COMMON_SETUP}
		SimpleOres2-1.16.3-2.3.0.2.jar Simple Ores {simpleores@1.16.3-2.3.0.2 COMMON_SETUP}
		forge-1.16.3-34.1.5-client.jar Minecraft {minecraft@1.16.3 COMMON_SETUP}
		useful_backpacks-1.16.3-1.11.5.82.jar Useful Backpacks {usefulbackpacks@1.11.5.82 COMMON_SETUP}
		DustrialDecor-1.0.6-1.16.3.jar 'Dustrial Decor {dustrial_decor@1.0.6 COMMON_SETUP}
		MouseTweaks-2.13-mc1.16.2.jar Mouse Tweaks {mousetweaks@2.13 COMMON_SETUP}
		assortedstorage-1.16.3-1.0.1.jar Assorted Storage {assortedstorage@1.16.3-1.0.1 COMMON_SETUP}
		BiomesOPlenty-1.16.3-12.0.0.400-universal.jar Biomes O' Plenty {biomesoplenty@1.16.3-12.0.0.400 COMMON_SETUP}
		valhelsia_structures-16.0.3.jar Valhelsia Structures {valhelsia_structures@16.0.3 COMMON_SETUP}
		LetSleepingDogsLie-1.16.3-1.1.1.jar Let Sleeping Dogs Lie {dogslie@1.1.1 COMMON_SETUP}
		SimpleCoreLib-1.16.3-2.0.0.5.jar SimpleCoreLib {simplecorelib@1.16.3-2.0.0.5 COMMON_SETUP}
		ornamental-1.16.3-4.2.jar Ornamental {ornamental@4.2 COMMON_SETUP}
		AdLods-1.16.3-4.0.2.0-build.0017.jar Large Ore Deposits {adlods@4.0.2.0 COMMON_SETUP}
		masonry-1.16.3-0.3.6.jar Masonry {masonry@NONE COMMON_SETUP}
		simplylight-1.16.3-1.0.2.jar Simply Light {simplylight@1.16.3-1.0.2 COMMON_SETUP}
		simplefarming-1.16.3-1.3.3.jar Simple Farming {simplefarming@1.16.3-1.3.3 COMMON_SETUP}
		xercamod-1.16.3-1.0.jar Xerca Mod {xercamod@1.16.3-1.0 COMMON_SETUP}
		WallpapercraftWalls-1.16.3-1.4.0.jar Wallpapercraft - Walls! {wallpapercraftwalls@1.4.0 COMMON_SETUP}
		collective-1.16.3-1.45.jar Collective {collective@1.45 COMMON_SETUP}
		DoubleSlabs-1.16-3.4.10.jar Double Slabs {doubleslabs@3.4.10 COMMON_SETUP}
		ftb-ultimine-1.1.0.10.jar FTB Ultimine {ftbultimine@1.1.0.10 COMMON_SETUP}
		buildersaddition-1.16.3-20200930a.jar Builders Crafts & Addition {buildersaddition@1.16.3-20200930a COMMON_SETUP}
		worldedit-forge-mc1.16.3-7.2.0-beta-05.jar WorldEdit {worldedit@7.2.0-beta-05+20b3da4 COMMON_SETUP}
		mcws-roofs-1.16.3-v2.0.0.jar Macaw's Roofs {mcwroofs@2.0.0 COMMON_SETUP}
		Aesthetics-1.16.3-1.10.0.0.jar Simple Ores: Aesthetics {aesthetics@1.16.3-1.10.0.0 COMMON_SETUP}
		WoV-1.1.4.jar War of Valor {shocked@1.16.3-1.1.4 COMMON_SETUP}
		ilikewood-1.16.3-4.0.2.2.jar I Like Wood {ilikewood@1.16.3-4.0.2.2 COMMON_SETUP}
		byg-1.0.7.jar Oh The Biomes You'll Go {byg@1.0.7 COMMON_SETUP}
		Aquaculture-1.16.3-2.1.1.jar Aquaculture 2 {aquaculture@1.16.3-2.1.1 COMMON_SETUP}
		AdChimneys-1.16.3-6.0.2.0-build.0017.jar Advanced Chimneys {adchimneys@6.0.2.0 COMMON_SETUP}
		terraqueous-1.16-1.7.3.jar Terraqueous {terraqueous@1.7.3 COMMON_SETUP, terraheads@1.0.0 COMMON_SETUP, pixelpowers@2.0.0 COMMON_SETUP}
		BetterMineshaftsForge-1.16.2-1.0.jar YUNG's Better Mineshafts {bettermineshafts@1.16.2-1.0 COMMON_SETUP}
	Kiwi Modules: 
		chineseworkshop:blocks
		chineseworkshop:debug_stick
		chineseworkshop:decorations
		chineseworkshop:materials
		chineseworkshop:retexture
		kiwi:contributors
		kiwi:data
	Launched Version: 1.16.3-forge-34.1.5
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: GeForce GTX 1060 6GB/PCIe/SSE2 GL version 4.6.0 NVIDIA 445.75, NVIDIA Corporation
	GL Caps: Using framebuffer using OpenGL 3.0
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'
	Type: Client (map_client.txt)
	Graphics mode: fast
	Resource Packs: 
	Current Language: English (US)
	CPU: 8x AMD FX-8370 Eight-Core Processor 
	OptiFine Version: OptiFine_1.16.3_HD_U_G3
	OptiFine Build: 20200916-082411
	Render Distance Chunks: 10
	Mipmaps: 0
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: null
	OpenGlVersion: 4.6.0 NVIDIA 445.75
	OpenGlRenderer: GeForce GTX 1060 6GB/PCIe/SSE2
	OpenGlVendor: NVIDIA Corporation
	CpuCount: 8

**Comment by Rukia1727** [comment]
Valhesia didn't cause it. For me at least it's dot hack that caused the crashes

**Comment by ShetiPhian** [comment]
Same Issue: #269

I'm wondering if this is a race condition.
I've been using the same reload listener as the default leaves (including error thrown on fail), what if something is causing a premature reload.

Give this build a try, I changed form using the Minecraft reload listener to one provided by Forge.
I also changed the error handling, so it shouldn't crash anymore just render tree leaves black if loading fails.
http://www.mediafire.com/file/as662vv9946fscs/terraqueous-1.16-1.7.4--TESTER1.jar/file
